shoop/admin/modules/orders/static_src/create/view/customers.js

```diff
@@ -40,6 +40,8 @@ function renderAddress(store, shop, customer, address, addressType) {
 export function customerSelectView(store) {
     const {customer, shop} = store.getState();
     return m("div.form-group", [
+        m("p", gettext("By not selecting existing customer new customer is created based on billing address.")),
```

Comment:
Maybe show this only if no customer is selected? :) Also, in that case would just

> A new customer will be created.

do?